libs/sdk-backend-bear/src/convertors/fromBackend/MetaConverter.ts

```diff
@@ -122,3 +123,9 @@ export const convertMetadataObjectXrefEntry = (
         deprecated: entry.deprecated === "1",
     };
 };
+
+export const convertUriIdentifierPair = (uriIdentifierPairs: IUriIdentifierPair): IUriIdentifierObject => {
+    return {
+        ...uriIdentifierPairs,
+    };
+};
```

Comment:
this code look wierd

Reply:
This just converts `IUriIdentifierPair` from `api-client-bear` to `IUriIdentifierObject` from `sdk-backend-spi`

Reply:
Why don't you reuse IUriIdentifierObject?